app/src/main/java/com/example/keniphhomemaintenance/dwellings/Dwelling.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.keniphhomemaintenance.dwellings
+
+class Dwelling(
+    val id: Int,
```

Comment:
What does the id represent?

Reply:
see comment in `Navigation.kt` line 23

Reply:
That doesn't tell me anything about what the id represents, it just says that an id exists. Where/how do we create the id? It looks like nowhere; when I search for usages of id there are none.

Reply:
id deleted as we don't need it for now